### PR TITLE
Docs-calicoctl-move

### PR DESCRIPTION
**Pull request by bmckercher123**
## Move calicoctl to /Operations from /Install

Approved by Casey
Merge to nightly

### CODE REVIEW

File: maintenance/clis/calicoctl/install.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+
+You can read more about API groups in [the Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/#api-groups).
+
+#### calicoctl and kubectl
+
+In order to manage {{site.prodname}} APIs in the `projectcalico.org/v3` API group, you should use `calicoctl`. This is because
+`calicoctl` provides important validation and defaulting for these resources that is not available in `kubectl`. However, `kubectl`
+should still be used to manage other Kubernetes resources.
+
+> **Note**: If you would like to use `kubectl` to manage `projectcalico.org/v3` API resources, you can try the
```
Comment (coutinhop) [inline maintenance/clis/calicoctl/install.md:32]: This paragraph changed to

```
> **Note**: If you would like to use `kubectl` to manage `projectcalico.org/v3` API resources, you can use the 
>           [Calico API server]({{site.baseurl}}/maintenance/install-apiserver).
{: .alert .alert-info}
```
in:

https://github.com/projectcalico/calico/blob/aceeaafdf97ca3ffb464ab77f7a3d8cf66ea4959/getting-started/clis/calicoctl/install.md

Could you make it reflect that here?

Comment (bmckercher123) [reply maintenance/clis/calicoctl/install.md:32]: Sure.

**Review by coutinhop — APPROVED** [review]
LGTM!

**Pull request merged**